python/pyproject.toml

```diff
@@ -36,7 +37,12 @@ tests = [
     "mypy~=1.5",
     "pytest~=8.0",
     "pytest-cov>=5.0,<7.0",
+    "runnb",
     "ruff~=0.9.0",
+    "nbformat",
+    "nbconvert",
+    "types-PyYAML",
```

Comment:
PyYAML is not used anymore, but rather ruamel.yaml, due to YAML formatting issues in PyYAML. So, installing the types seems not necessary.